app/src/main/java/org/mozilla/focus/home/NavigationOverlayAnimations.kt

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.home
+
+import android.animation.Animator
+import android.animation.AnimatorListenerAdapter
+import android.animation.AnimatorSet
+import android.animation.ObjectAnimator
+import android.support.v4.view.animation.FastOutSlowInInterpolator
+import android.view.View
+import kotlinx.android.synthetic.main.fragment_navigation_overlay.view.*
+import org.mozilla.focus.ext.onGlobalLayoutOnce
+
+/**
+ * Encapsulation of animation code for the navigation overlay.
+ */
+object NavigationOverlayAnimations {
+
+    fun onCreateViewAnimateIn(overlay: View, isInitialHomescreen: Boolean) {
+        if (isInitialHomescreen) return
```

Comment:
If this is true, it would mean we're in a pretty inconsistent state.  Perhaps worth adding telemetry here?

Reply:
This is called every time the app starts. I expressing "we don't animate on the initial homescreen" was much clearer to do here, in the animation file, than it was to do it in the overlay itself. Additionally, it keeps the overlay onCreateView logic higher level.